### PR TITLE
Project MVP 

### PR DESCRIPTION
**Pull request by AdnanCodes**

### CODE REVIEW

File: src/00_hello.py [hunk 1]
```diff
@@ -1 +1,2 @@
-# Print "Hello, world!" to your terminal
\ No newline at end of file
+# Print "Hello, world!" to your terminal
+print("Hello, world!")
```
Comment (brianilles) [inline src/00_hello.py:2]: Great code. It all starts with a hello world :)

File: src/01_bignum.py [hunk 2]
```diff
@@ -1,4 +1,9 @@
 # Print out 2 to the 65536 power
 # (try doing the same thing in the JS console and see what it outputs)
 
-# YOUR CODE HERE
\ No newline at end of file
+# YOUR CODE HERE
+
+print(2 ** 25536)
```
Comment (brianilles) [inline src/01_bignum.py]: What happens in the JS console?

**Review by brianilles — COMMENTED** [review]
Great start to the two day project!

File: src/01_bignum.py (hunk 2 above)
Comment (AdnanCodes) [reply src/01_bignum.py]: Oh, I forgot to write it there but it shows Inifinity in JS console.

File: src/04_printing.py [hunk 3]
```diff
@@ -11,7 +11,8 @@
...
 # x is 10, y is 2.25, z is "I like turtles!"
-
+print("x is %s,y is %s, z is %s" % (x, y, z))
 # Use the 'format' string method to print the same thing
-
-# Finally, print the same thing using an f-string
\ No newline at end of file
+print('x is {0}, y is {1}, z is {2}'.format(x, y, z))
+# Finally, print the same thing using an f-string
+print(f'x is {x},y is {y}, z is {z!r}')
```
Comment (brianilles) [inline src/04_printing.py:18]: Great work with these three. Which implementation is your fav? 
Cool use of the `!r` here. Its a clear example of where `!r` applies `repr()` which converts the value before its formatted thereby leaving the quotes in the final output. NEAT!

File: src/06_tuples.py [hunk 4]
```diff
@@ -17,28 +17,33 @@
...
 
 
-
 # Write a function `print_tuple` that prints all the values in a tuple
 
 # YOUR CODE HERE
+def print_tuple(items):
+    for item in items:
+        print(item)
+
```
Comment (brianilles) [inline src/06_tuples.py:42]: Clean and readable! I appreciate you using `item` as your traversal variable as opposed to `i`, for example.

File: src/07_slices.py [hunk 5]
```diff
@@ -12,26 +12,26 @@
 a = [2, 4, 1, 7, 9, 6]
 
 # Output the second element: 4:
-print()
+print(a[slice(1, 2)])
```
Comment (brianilles) [inline src/07_slices.py:16]: Nice work here. Please note that the same can be accomplished with `a[1]` and similar syntax with the rest of these in the file.

File: src/08_comprehensions.py [hunk 6]
```diff
@@ -27,15 +29,19 @@
...
 # the user entered into list x.
 
 x = input("Enter comma-separated numbers: ").split(',')
-
+print(x)
 # What do you need between the square brackets to make it work?
 y = []
+for even in x:
+    if int(even) % 2 == 0:
+        y.append(even)
```
Comment (brianilles) [inline src/08_comprehensions.py:46]: Good stuff. How would you refactor these to be List Comprehensions?

File: src/10_functions.py [hunk 7]
```diff
@@ -2,11 +2,17 @@
...
 # YOUR CODE HERE
 
+
+def checkEven(num):
+    if num % 2 == 0:
+        print("Even")
+    else:
+        print("Odd")
+
+
```
Comment (brianilles) [inline src/10_functions.py:12]: Exactly! Here, we can save 3 lines of code by simply writing `return n % 2 == 0`  which will return a Boolean in it of itself. Honestly though, sometimes I find the way you did it to be more readable. However I thought I'd mention it so you are aware if you were not already!

File: src/13_file_io.py [hunk 8]
```diff
@@ -9,10 +9,16 @@
 # Print all the contents of the file, then close the file
 
 # YOUR CODE HERE
+with open('foo.txt') as f:
+    read_data = f.read()
+    print(read_data)
```
Comment (brianilles) [inline src/13_file_io.py:15]: Great use of `with open()`. I pretty much use this exclusively for file IO as the file doesn't need to be explicitly closed.

**Review by brianilles — APPROVED** [review]
Great work Adnan! I can see you have quickly yet solidly found a base of knowledge within Python in these first two days. Can't wait to see your work for the rest of the week! I will approve and close this PR. Superb!

**Pull request merged**